rewards/utils/emission_utils.py

```diff
@@ -32,6 +33,39 @@ def get_flat_emission_rate(sett: str, chain: str) -> Decimal:
     return Decimal(get_emission_control(chain).proRataEmissionRate(sett).call() / 1e4)
 
 
+@lru_cache
```

Comment:
Is this cache going to solve any problems? I think caching should be used VERY carefully, can create lots of problems in the future that will be super hard to debug

Reply:
Well the cache is this case is created so that we don't have to recreate the contract object every time that we call the get_nft_weight function. We are using this for most of the contracts but its fine if we get rid of them the slowdown shouldn't be that bad

Reply:
Ok, let's leave the cache for the deployed contract then 👍